test/move_effect_revival_blessing.c

```diff
@@ -0,0 +1,109 @@
+#include "global.h"
+#include "test_battle.h"
+
+#define MOVE_MESSAGE(name)                       \
+    if (B_EXPANDED_MOVE_NAMES == FALSE)          \
+        MESSAGE(#name" used RevivlBlesng!");     \
+    else                                         \
+        MESSAGE(#name" used Revival Blessing!"); \
+
+ASSUMPTIONS
+{
+    ASSUME(gBattleMoves[MOVE_REVIVAL_BLESSING].effect == EFFECT_REVIVAL_BLESSING);
+}
+
+TO_DO_BATTLE_TEST("Revival Blessing revives a chosen fainted party member for the player");
+// SINGLE_BATTLE_TEST("Revival Blessing revives a chosen fainted party member for the player")
+// {
+//     GIVEN {
+//         PLAYER(SPECIES_WOBBUFFET);
+//         PLAYER(SPECIES_WYNAUT) { HP(0); }
+//         OPPONENT(SPECIES_WOBBUFFET);
+//     } WHEN {
+//         TURN { MOVE(player, MOVE_REVIVAL_BLESSING); SEND_OUT(player, 1); }
+//     } SCENE {
+//         MOVE_MESSAGE(Wobbuffet)
+//         MESSAGE("Wynaut was revived and is ready to fight again!");
+//     } FINALLY {
+//         EXPECT_NE(GetMonData(&gPlayerParty[1], MON_DATA_HP), 0);
+//     }
+// }
+
+SINGLE_BATTLE_TEST("Revival Blessing revives the first fainted party member for an opponent")
+{
+    GIVEN {
+        PLAYER(SPECIES_WOBBUFFET);
+        OPPONENT(SPECIES_RAICHU);
+        OPPONENT(SPECIES_PICHU) { HP(0); }
+        OPPONENT(SPECIES_PIKACHU) { HP(0); }
+    } WHEN {
+        TURN { MOVE(opponent, MOVE_REVIVAL_BLESSING); }
+    } SCENE {
+        MOVE_MESSAGE(Foe Raichu)
```

Comment:
I think it would be nicer/more idiomatic if the usage of this macro looked more like `MOVE_MESSAGE("Foe Raichu");`. All you'd need to do is lose the `#` before `name`, and wrap the body of the macro in a `do {` … `while (0)`.